Review commit

bpankowski opened the pull request.

bpankowski committed:
Review commit

jluk-codi commented on requirements.txt:
```diff
@@ -0,0 +1,2 @@
+pkg-resources==0.0.0
```
It breaks on my setup:
```[root@500aefcf1c74 host]# pip install -r requirements.txt 
Collecting pkg-resources==0.0.0 (from -r requirements.txt (line 1))
  Could not find a version that satisfies the requirement pkg-resources==0.0.0 (from -r requirements.txt (line 1)) (from versions: )
No matching distribution found for pkg-resources==0.0.0 (from -r requirements.txt (line 1))
You are using pip version 8.1.2, however version 18.0 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.```

jluk-codi commented on jira-doc.py:
```diff
@@ -0,0 +1,137 @@
+from jira import JIRA
+from datetime import datetime, timedelta
+import json
+import ast
```
unused import

jluk-codi commented on jira-doc.py:
```diff
@@ -0,0 +1,137 @@
+
+def get_list(sprint):
+    query_string = ('project=JD and status="Done" and Sprint="' + sprint +'"')
+    old_sprint = (int(sprint)-1)
```
this may be a sprint from another project :( You need to find a more reliable way of determining the previous sprint in JD

jluk-codi commented on jira-doc.py:
```diff
@@ -0,0 +1,137 @@
+status = ''
+empty_list = []
+# print (check_sprint)
+for x in check_sprint:
```
Rewrite this as a "get current sprint" function, return the sprint Id, and call the `get_list` function afterwards. It will be easier to understand the flow

jluk-codi commented on jira-doc.py:
```diff
@@ -0,0 +1,137 @@
+    inprogress = jira.search_issues(query_string)
+    print ('In progress')
+    file.write('In progress \n')
+    for x in inprogress:
```
remove repetition of these loops (todo, in progress, done), extract them to a separate function and call with parameters.

jluk-codi commented on jira-doc.py:
```diff
@@ -0,0 +1,137 @@
+    print ('To do')
+    file.write('To do \n')
+    for x in todo:
+        # update_date = x.raw['fields']['updated']
```
Clean up unneeded commented code (here and in other places).

jluk-codi reviewed (changes requested):
Remove trailing spaces from the file.

bpankowski committed:
Deleted unwanted comments, packed into function code
wich is checking current sprint. Added to gitignore file venv env files.
Changed requirements file.
